numpy/f2py/tests/src/parameter/constant_array.f90

```diff
@@ -0,0 +1,45 @@
+! Check that parameter arrays are correctly intercepted.
+subroutine foo_array(x, y, z)
+  implicit none
+  integer, parameter :: dp = selected_real_kind(15)
+  integer, parameter :: pa = 2
+  integer, parameter :: intparamarray(2) = (/ 3, 5 /)
+  integer, parameter, dimension(pa) :: pb = (/ 2, 10 /)
+  integer, parameter, dimension(intparamarray(1)) :: pc = (/ 2, 10, 20 /)
```

Comment:
Is it legal to have these in a different order like this?
```suggestion
  integer, dimension(intparamarray(1)), parameter :: pc = (/ 2, 10, 20 /)
```
If so, can we add a test for that too?

Reply:
yes, the order of attribute specifications is insignificant.